### DISCUSSION TITLE
How to use it effectively on OSX?

### DISCUSSION THREAD

**Discussion by vladcebo** [VSCodeVim/Vim#8683] [General]
Hello,

So I have a linux PC but at work I'm using a MacBook. For OSX I have remapped the keyboard so Ctrl and Cmd are switched places, thus most of the shortcuts (such copy, paste, VSCode ones) have the same like "Ctrl" layout as on Linux. Now that's ok, but when using VSCodeVim,  it's still Ctrl, which now is in place of command. Is it possible to override globally to use command instead of ctrl?

**Comment by ianschmitz** [comment]
I'm not sure how this question is related to the vim extension, but you might want to check out https://karabiner-elements.pqrs.org/ which can do complex keybindings, and can do so depending on the active program.

**Comment by lamhungypl** [comment]
I'd recommend you try remapping Ctrl and Caplock. Cmd-related shortcuts are reasonable in Mac layouts IMO
